unit_test/tstForce.hpp

```diff
@@ -379,7 +379,7 @@ computeReferenceForceX( QuadraticTag,
 //---------------------------------------------------------------------------//
 template <class ModelType>
 CabanaPD::Particles<TEST_MEMSPACE, typename ModelType::base_model,
-                    typename ModelType::thermal_type>
+                    typename ModelType::thermal_type, CabanaPD::OutputEnergy>
```

Comment:
Is the default case of `NoOutput` explicitly tested?

Reply:
It's not, are you thinking of a write/read test for all the relevant fields? We're very light on unit tests across the board..

Reply:
No, I was just thinking from a coverage POV to test all the possible template parameter instantiations. A heavier lift for sure, but could be a separate issue for later.

Reply:
Opened #156